server/enterprise/message_export/shared/export_data.go

```diff
@@ -273,6 +289,8 @@ func getJoinsAndLeaves(startTime int64, endTime int64, channelMembersHistory []*
 	type StillJoinedInfo struct {
```

Comment:
Could this be `StillMember`? Or similar?

Reply:
Sure, sounds good!